### PR TITLE
Development update

### PR DESCRIPTION
**Pull request by pgunnell**
Some proposal of modifications (first attempt: target is to present on Wednesday the new features and get feedback):

- update_twiki to create a database for what is contained in the twiki from Phat (if approved, update_data should be also modified)
- inclusion of notes text field on the sample page
- rearrangement of the index layout, including only campaign groups
- inclusion of more campaigns

Feedback on coding style very welcome ;)

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE HTML>
 <html>
   <head>
-    <base href="/samples/">
+    <base href="">
```
Comment (justinasr) [inline html/index.html]: Please revert back to `href="/samples/"` as this will be available at ....cern.ch/samples/

File: html/index.html [hunk 2]
```diff
@@ -21,26 +21,15 @@ <h5>Existing samples</h5>
...
-        </tr>
         {% for row in campaigns %}
-          <tr>
             <!-- Campaign group -->
             {% if row[0][1] > 0 %}
-            <td rowspan={{row[0][1]}}>
+            <th rowspan={{row[0][1]}}>
               <a href="campaign_group/{{row[0][0]}}">{{row[0][0]}}</a>
-            </td>
+            </th>
```
Comment (justinasr) [inline html/index.html]: Remove trailing spaces and tabs

File: update_twiki.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                  total_events integer NOT NULL,
+                  campaign text NOT NULL,
+                  resp_group text NOT NULL)''')
+
+    # Clear the table
+    c.execute('DELETE FROM `twiki_samples`')
+    conn.commit()
+
+    # Get all needed requests
+    twiki_samples_fall_18_candidates = mcm.get('requests', query='member_of_campaign=RunIIFall18pLHE')
```
Comment (justinasr) [inline update_twiki.py]: You probably want to allow different campaigns to be used here. Remove RunIIFall18pLHE and add either a list or command line argument. Best option - add a command line argument that has comma separated campaign names

File: update_data.py [hunk 4]
```diff
@@ -336,8 +359,7 @@ def process_request(request, campaign, cursor, table):
 
 conn.commit()
 
-
-# TWiki samples
+# TWiki samples - old mode
 with open('MainSamplesFall18.txt') as f:
```
Comment (justinasr) [inline update_data.py]: If you are using `update_twiki.py` then this can be removed

Comment (pgunnell) [reply update_data.py]: Thanks! I think it's not ready yet for the change, so I will leave it as it is for now.

**Pull request merged**